### ISSUE TITLE
Some archive pages don't have a main heading

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-showcase-2022#180]
The design shows no main header on the page, but it's required for accessibility and probably SEO, so there's a `query-title` block hidden with the `screen-reader-text` class.

It's present on category archives: https://wordpress.org/showcase-v2/category/business/

But not present when the filters are used: https://wordpress.org/showcase-v2/archives/?tag%5B%5D=business

I think we can hook into the query-title block's render output to make sure it's always outputting a title.

To make this a little less abstract, here's an example with the title visible:

![Screen Shot 2023-09-25 at 18 43 02](https://github.com/WordPress/wporg-showcase-2022/assets/541093/6b5045fa-3574-4ab4-a647-6b73ef309f85)

What should that title be on pages with complex filtering? e.g., "[Category: Featured, Tag: Fashion](https://wordpress.org/showcase-v2/archives/?cat%5B%5D=214&tag%5B%5D=fashion)" or "[Search: news, Category: Featured](https://wordpress.org/showcase-v2/archives/?cat[]=343&s=news)" — maybe @eidolonnight @thetinyl can help?

There's also a possibility that the title should be visible after all, which will likely need a design, cc @WordPress/meta-design

**Comment by jasmussen** [comment]
I'm going to assume that the title won't need to be visible, as the filter bar above is meant to be the visual indication of what you see below, and thus serve the role. If that is an incorrect assessment we'll cross the road when we get there.

For the invisible heading, and related to this:

> What should that title be on pages with complex filtering? e.g., "[Category: Featured, Tag: Fashion]"

I think that's fine, though I'd omit the brackets.

**Comment by thetinyl** [comment]
> I think that's fine, though I'd omit the brackets. We could also simplify it further, perhaps "Featured, Business, and Fashion", or "Featured and Fashion".

I agree with this. I don't think it adds anything to specify if something is a category vs. tag in this use case. 

Going to do the writer thing and throw out a suggestion to add more words... What about adding a label or something like: "Filtering by: Featured, Business, and Fashion" ...could help contextualize further.

**Comment by eidolonnight** [comment]
Along the lines of providing context, I wonder if it's possible to show the category descriptions? It could be a fun way to explain the category and what WordPress has to offer.

**Comment by ryelle** [comment]
> something like: "Filtering by: Featured, Business, and Fashion"

Sounds good to me.

> Along the lines of providing context, I wonder if it's possible to show the category descriptions?

We could, but it would only show when viewing a single category archive. That would also need design input along with showing the heading.

**Comment by ryelle** [comment]
I'm going to move forward with fixing the missing headers using the phrasing suggested for now. Making the heading visible can be handled in another issue once there's a design for it.
